src/lib/Libpython/pbs_python_external.c

```diff
@@ -249,17 +249,19 @@ pbs_python_ext_start_interpreter(
 	}
 
 	interp_data->pbs_python_types_loaded = 1; /* just in case */
-	return;
-
+SUCCESS_EXIT:
+	return 0;
 ERROR_EXIT:
-	pbs_python_ext_shutdown_interpreter(interp_data);
-	return;
+	if (interp_data->interp_started) {
```

Comment:
Styling: no need to surround single line if blocks with curly braces (Ref: https://pbspro.atlassian.net/wiki/spaces/DG/pages/5537826/Coding+Standards)